src/java.base/unix/classes/java/lang/ProcessEnvironment.java

```diff
@@ -128,10 +137,19 @@
             theOriginalUnmodifiableEnvironment = null;
             theEnvironment = origEnvironment;
         }
+/*[ELSE] CRIU_SUPPORT*/
+            theEnvironment.put(Variable.valueOf(environ[i-1]),
```

Comment:
the indentation is a bit off here

Reply:
It matches the `master` branch (OpenJDK content) which follows `for (int i = environ.length-1; i > 0; i-=2)`, hope this avoids potential merge conflict.

Reply:
Please keep the braces (`{` & `}`) around the `for` body in this section.

Reply:
This intended to not modify OpenJDK code https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/0d08c7e9d73ad328724c60d34bb954159d1497b1/src/java.base/unix/classes/java/lang/ProcessEnvironment.java#L74-L76

Reply:
> The braces are there upstream;

can you provide a link to the code snippet?

Reply:
Ok, sorry, my mistake. The braces I was expected to see were added by you, @JasonFengJ9, in 6f41d57649e995ad221d827388fdf88be275a0b6.